bootstrap-scripts/pico/kafka.sh

```diff
@@ -25,6 +25,7 @@ cat >> /etc/salt/grains <<EOF
   - platform_testing_general
   - kafka_manager
   - elk
+  - kafka_tool
```

Comment:
no need to install kafkat as part of kafka instance, let's keep it only on the tool node

Reply:
In pico kafka-manager is installed only in kafka node not on edge.